README.rst

```diff
@@ -141,6 +141,34 @@ Use storage on any of the many cloud providers `rclone <https://rclone.org/>`_ s
 - implementation of this primarily depends on the specific remote.
 
 
+s3
+~~~
```

Comment:
If the underlining is longer than the headline text, does the rst parser / renderer process this correctly?

Reply:
Seems so

Reply:
Made the change. Looks ok.